.github/workflows/basic.yml

```diff
@@ -42,17 +43,23 @@ jobs:
             compiler: {c: gcc, cxx: g++}
             shared_library: 'OFF'
             os_provider: 'OFF'
+        exclude:
+          # TODO: Remove this exclusion. There's a leak reported by sanitizer
```

Comment:
please fill an issue in the github to fix this

Reply:
This is the root cause of this issue: https://github.com/oneapi-src/unified-memory-framework/blob/main/src/provider/provider_tracking.c#L70-L80

Reply:
Issue: https://github.com/oneapi-src/unified-memory-framework/issues/122